### PR TITLE
Tabbed navigation a11y fixes

### PR DESCRIPTION
**Pull request by eric-gade**
## What does this PR do? 🛠️
This PR addresses #733 by adding the expected accessibility behavior and attributes as [recommended by the W3C per these guidelines](https://www.w3.org/WAI/ARIA/apg/patterns/tabs/examples/tabs-manual/).

## What does the reviewer need to know? 🤔
Appropriate ARIA attributes and relationships have been added where needed, and we've added a few e2e tests to make sure the navigation that W3C recommends is implemented properly. Specific nav things to look out for:
* If a tab is in focus, pressing the `Tab` key __should not__ move to the next tab button, but rather put the focus into the corresponding tabpanel (ie the tab's content container).
* Left/right arrow keys are used to put the focus on the next or previous tab button, if a tab button is currently in focus (instead of the `Tab` key, see above)
* Left/right arrows that go too far in either direction should simply cycle around to the first/last tab button as needed
* Home key puts focus on the first tab button
* End key puts focus on the last tab button

## Testing
__Please__ be sure the a11y test this locally using VoiceOver or Accessibility Inspector!

### CODE REVIEW

File: web/themes/new_weather_theme/templates/layout/page--local.html.twig [hunk 1]
```diff
@@ -133,35 +133,36 @@
 
         {{ drupal_block("weathergov_current_conditions") }}
 
-      <tabbed-nav class="display-block position-relative">
-        <div class="tab-buttons padding-bottom-4 padding-top-4 top-0 z-top display-flex flex-row flex-justify position-sticky bg-white top-4 mobile-lg:grid-col-8 desktop:grid-col-6">
+        <tabbed-nav class="display-block position-relative">
+          <div id="tablist-label" class="usa-sr-only">{{ "Weather information sections" | t }}</div>
```
Comment (eric-gade) [inline web/themes/new_weather_theme/templates/layout/page--local.html.twig:137]: @partly-igor Note this text here, which we talked about but perhaps didn't finalize

Comment (partly-igor) [reply web/themes/new_weather_theme/templates/layout/page--local.html.twig:137]: This is good!

**Review by greg-does-weather — APPROVED** [review]
This is excellent. One non-blocking comment.

## Code Review Checklist

A checked item is either an assertion that I have tested this item or an indication that I have verified it does not apply to this pull request.

## The Basics
- [x] Checks are passing
- [x] I read the code
- [x] I ran the code
- [x] (if applicable) Post deploy steps are run
- [x] (if applicable) I validated the change on the deployed version in <environment>

## Documentation
- [x] changes to “how we do things” are documented in READMEs
- [x] all new functions and methods are commented using plain language
- [x] any new modules added documented in modules.md

## Security
- [x] security false positives are documented
- [x] data from external sources is cleaned and clearly marked

## Reliability
- [x] error handling exists for unusual or missing values
- [x] interactions with external systems are wrapped in try/except
- [X] functionality is tested with unit or integration tests
- [X] dependency updates in composer.json also got changed in composer-lock.json

## Infrastructure
- [x] all changes are auditable and documented via a script
- [x] it is clear who can and should run the script
- [x] (if applicable) diagrams have been updated or added in PlantUML

## Accessibility
- [x] New pages have been added to cypress-axe file so that they will be tested with our automated accessibility testing
- [x] Meets WCAG 2.0 AA or 2.1 AA for Section 508 compliance
    - [x] Site is keyboard accessible. All interactions can be accessed with a keyboard
    - [x] Site is free of keyboard traps. The keyboard focus is never trapped in a loop
    - [ ] All form inputs have explicit labels
    - [ ] Form instructions are associated with inputs
    - [x] All relevant images use an img tag
    - [x] All images have appropriate alt attributes
    - [x] Multimedia is tagged. All multimedia has appropriate captioning and audio description
    - [x] Text has sufficient color contrast. All text has a contrast ratio of 4.5:1 with the background
    - [x] Site never loses focus. Focus is always visible when moving through the page with the keyboard
    - [x] Tab order is logical
    - [x] Tables are coded properly. Tables have proper headers and column attributes
    - [x] Headings are nested properly. Heading elements are nested in a logical way
    - [x] Language is set. The language for the page is set
    - [x] CSS is not required to use the page. The page makes sense with or without CSS
    - [x] Links are unique and contextual. All links can be understood taken alone, e.g., ‘Read more - about 508’
    - [x] Page titles are descriptive

## Device Matrix
- [x] firefox/gecko (renders correctly and user interactions work)
- [x] chrome/chromium/edge (renders correctly and user interactions work)
- [x] safari/webkit (renders correctly and user interactions work)
- [x] web page is readable and usable 
    - [x] at 480px     (mobile)
    - [x] at 640px     (tablet)
    - [x] at 1024px    (desktop)

**Comment by partly-igor** [comment]
I tested with VO and keyboard and it was working great – only question I had was about the home/end presses, should we also prevent default on that? Right now it’ll both switch the tabs and scroll to top/bottom.

**Comment by partly-igor** [comment]
I'll note from my conversation with Eric I don't have actual `home` and `End` keys, but instead I press `Fn + Arrow` on a mac keyboard, so it may not be exactly that.

**Pull request merged**